lips/lip-3.md

```diff
@@ -7,3 +7,82 @@ type: Informational
 created: 06/09/2020
 ---
 
+# Introduction
+Libra Upgrade Management (LUM) is a set of protocols for the Libra Networks company to reconfigure the blockchain. With respect to time-sensitive decisions and incident response, LUM provides the Libra Networks--a subsidiary of the Libra Association licensed in Switzerland to operate a payment system--with technical means to respond to critical incidents, as (i) necessitated by its license terms and the law and (ii) as determined by the Libra Network risk appetite/tolerance policy.
+
+With respect to complex technical matters, LN can use the TSC, or some other small group, as "advisors" as to what path to take.
```

Comment:
I would remove this sentence as it is up to LN's discretion on how to leverage the LUM protocols and this is a technical LIP, not a governance one.